### ISSUE TITLE
Social Logins 

### ISSUE THREAD

**Issue by tylerpashigian** [EveripediaNetwork/wagmi-magic-connector#49]
- **I'm submitting a**
  [ ] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [X] question about how to use this project

- **Summary**
When logging in via the social logins (Google) in the `MagicAuthConnector`, my page is reloading when coming back from the Google SSO auth flow. Despite having `autoConnect` set to true on my wagmi connector, the address and isConnected are coming back as `undefined` and `false`, respectively. When clicking the button that triggers my `MagicAuthConnector` again, my `account` and `isConnected` are set correctly without going back into the SSO flow. This does **not** happen when connecting via email (I assume since the page doesn't reload?). Does anyone have any idea how to ensure these values are set correctly when coming back from the google SSO flow?

- **Version**
  - @everipedia/wagmi-magic-connector version: `^0.9.1`
  - wagmi version: `^0.9.0` (cant upgrade, limited because rainbowkit has issues with the latest version of wagmi)

**Comment by Royal-lobster** [comment]
Is this still a issue with `@everipedia/wagmi-magic-connector@0.12.1` ?

**Comment by tylerpashigian** [comment]
> Is this still a issue with `@everipedia/wagmi-magic-connector@0.12.1` ?

Yeah, I just upgraded and I still seem to have to trigger the login flow again in order to get wagmi to detect I am connected/authenticated. What are the dependency versions? (ex. wagmi, ethers, etc). Im wondering if I also need to get upgrade those versions in order for the updates to reflect correctly.

**Comment by Royal-lobster** [comment]
> > Is this still a issue with `@everipedia/wagmi-magic-connector@0.12.1` ?
> 
> Yeah, I just upgraded and I still seem to have to trigger the login flow again in order to get wagmi to detect I am connected/authenticated. What are the dependency versions? (ex. wagmi, ethers, etc). Im wondering if I also need to get upgrade those versions in order for the updates to reflect correctly.

Oh yes, wagmi needs to be latest `1.0.5`, and the package no longer needs ethers, instead it relies on viem. please check out the migration guide at wagmi, https://wagmi.sh/core/migration-guide
